lessons/githubPages/lesson.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: page
+title: 'Web design with Jekyll/Github Pages'
+visible: false
+---
+<!-- change visible to true if you want it on the site -->
+
+ - **Authors**: Dawn Walker [@dcwalk](https://github.com/dcwalk)
```

Comment:
yeah, I always assume it won't work so treat it like any other link...will leave as is!